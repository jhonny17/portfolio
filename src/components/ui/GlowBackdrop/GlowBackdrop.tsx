import cx from 'classnames';

export type GlowBackdropProps = {
  className?: string;
};

export const GlowBackdrop = ({ className }: GlowBackdropProps) => {
  return (
    <div
      className={cx(
        'z-[-1]',
        'absolute',
        'inset-0',
        'blur-3xl',
        'bg-gradient-to-r',
        'from-emerald-900',
        'from-0%',
        'to-emerald-800',
        'to-100%',
        className,
      )}
    />
  );
};
