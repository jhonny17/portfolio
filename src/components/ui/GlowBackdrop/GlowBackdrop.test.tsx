import { expect, it } from 'vitest';
import { render } from '@testing-library/react';

import { GlowBackdrop } from './GlowBackdrop';

it('matches snapshot', () => {
  const { asFragment } = render(<GlowBackdrop />);
  expect(asFragment()).toMatchSnapshot();
});
