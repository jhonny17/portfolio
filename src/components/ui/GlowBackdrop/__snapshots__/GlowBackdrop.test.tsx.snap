// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="z-[-1] absolute inset-0 blur-3xl bg-linear-to-r from-emerald-900 from-0% to-emerald-800 to-100%"
  />
</DocumentFragment>
`;
